internal/commands/push/inputs.go

```diff
@@ -57,6 +59,16 @@ func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 			return errProjectNotFound{}
 		}
 		i.LocalPath = app.RootDir
+	} else {
+		_, err := os.Stat(path.Join(profile.WorkingDirectory, i.LocalPath))
```

Comment:
[thought] I think we can tweak this approach a bit (and change a bit of the existing implementation).  we could check this (on `searchPath`) before we attempt to `local.LoadAppConfig`, and return the proper error if the directory doesn't exist.

then we `local.LoadAppConfig` as normal.

then we can move the `if app.RootDir == ""` outside of its enclosing `if i.LocalPath == ""` check to be right after `local.LoadAppConfig`, and return the other error type if `RootDir` is empty.